wca-guidelines.md

```diff
@@ -139,6 +139,8 @@ To be more informative, each Guideline is classified using one of the following
 
 - 11e+) [CLARIFICATION] Since an appeal is not guaranteed to be successful, the competitor may choose to keep the timer running while appealing it, and resume the attempt when appropriate.
```

Comment:
```suggestion
- 11e+) [CLARIFICATION] Since an appeal is not guaranteed to be successful, the competitor may choose to keep the timer running while appealing it, and resume the solve when appropriate.
```